include/irr/asset/filters/CBasicImageFilterCommon.h

```diff
@@ -51,9 +51,10 @@ class CBasicImageFilterCommon
 
 		struct default_region_functor_t
 		{
-			inline bool operator()(IImage::SBufferCopy& newRegion, const IImage::SBufferCopy* referenceRegion) { return true; }
+			constexpr default_region_functor_t() = default;
+			inline constexpr bool operator()(IImage::SBufferCopy& newRegion, const IImage::SBufferCopy* referenceRegion) const { return true; }
 		};
-		static default_region_functor_t default_region_functor;
+		_IRR_STATIC_INLINE_CONSTEXPR default_region_functor_t default_region_functor{};
```

Comment:
static inline `const`, not `constexpr`

Reply:
why?

Reply:
because its a non trivial struct, I believe C++ standard states that only trivial PoD can be constexpr

Reply:
any literal type can be constexpr. And literal type is any type with at least one constexpr constructor. But anyway this `default_region_functor` somewhere, idk what happened with it